### ISSUE TITLE
Update New JIRA Issue Description for Keptn Evaluation Done events

### ISSUE THREAD

**Issue by robertjahn** [keptn-sandbox/jira-service#16]
Here is the proposed template

Description =

project: {project}
service: {service}
stage: {stage}
result: {result}
score: {score}

start: {start}
end: {end}
strategy: {strategy}

Keptn Bridge: {Bridge URL}

Here is an example

Project: bitbucket-demo
Service: simplenodeservice
Stage: staging
result: pass
score: 100
 
Start: 2020-04-13T18:14:38Z
End: 2020-04-13T18:15:45Z
Strategy: Manual
 
Keptn Bridge: https://bridge.keptn.KEPTN-DOMAIN/project/bitbucket-demo/simplenodeservice/f00079b9-c94c-484d-847c-9158ceef8b6c/a0d83a6f-2462-485c-b98f-c03856bc403b

**Comment by ghost** [comment]
Does this help?
![image](https://user-images.githubusercontent.com/13639658/80559377-aa5c8c80-8a20-11ea-8be6-350accf2e6c5.png)

**Comment by robertjahn** [comment]
nice job!    I tested it out and the JIRA label with PASS/FAIL is working now too.
 
1) looks like an issue though with the start/stop times.  Looks like you are using END TIME for both START and END.
2) What if you made title with [KEPTN EVALUATION] versus just [EVALUATION]
3) how hard is it to add color/bold? example:
![image](https://user-images.githubusercontent.com/24907089/80605198-c05a5500-8a00-11ea-8dcf-2f069fb86331.png)
4) do we need both LINK in the description and the WebLink?  If I had to choose one, I would use the weblink since you get the logo
